tools/docker/corda-all-in-one/corda-v5/cordapps-deployment/healthcheck.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
Same as the previous review point. Lets have just one set of Dockerfile, Readme, healthcheck.. (which should be this, the one you added in this PR)